### PR TITLE
Add Travis-CI configuration

### PR DESCRIPTION
**Pull request by flimzy**
If nobody objects to Travis-CI for this repo, instead of Circle-CI, this should be ready to merge.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+all: test
+
+test:
+	for f in $(GO_FILES); do \
+		if [ -n "`gofmt -l $$f`" ]; then \
+			echo "$$f is not gofmted"; \
+			exit 1; \
+		fi; \
+	done
```
Comment (shurcooL) [inline Makefile]: Consider an alternative implementation:

```bash
diff -u <(echo -n) <(gofmt -d ./)
```

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+go:
+    - 1.5
+
+addons:
+    apt:
+        packages:
+            - nodejs
+install:
+    - go get -u github.com/gopherjs/gopherjs
```
Comment (shurcooL) [inline .travis.yml]: I would prefer if the `.travis.yml` formatting were consistent with some existing ones. E.g., see https://github.com/gopherjs/snippet-store/blob/master/.travis.yml.

Notably, the difference is using 2 spaces for indentation, and not having blank lines.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+	gopherjs test
```
Comment (shurcooL) [inline Makefile]: This `Makefile` can be avoided if you add `diff -u <(echo -n) <(gofmt -d ./)` and `gopherjs test` directly to `.travis.yml` under `script:` section, leading to a simpler repository.

**Comment by shurcooL** [comment]
I don't mind Travis CI, I use it myself in all my repos.

I've suggested some changes to simplify the setup. I think https://github.com/gopherjs/snippet-store/blob/master/.travis.yml is a good example to follow. But it's up to you if you want to apply these suggestions.

LGTM.

**Comment by flimzy** [comment]
Thanks for the suggestions. Much appreciated. I believe I have incorporated all of them.

**Comment by shurcooL** [comment]
Yep, thanks!

**Pull request merged**